build.gradle

```diff
@@ -63,28 +63,17 @@ task wrapper(type: Wrapper) {
 	gradleVersion = gradleWrapperVersion
 }
 
-grailsPublish {
-	user = System.getProperty("BINTRAY_USER")
-	key = System.getProperty("BINTRAY_KEY")
-	githubSlug = 'grails-plugins/grails-spring-security-oauth2'
-	license {
-		name = 'Apache-2.0'
+publishing {
+	repositories {
+		maven {
+			name = "GitHubPackages"
+			url = "https://maven.pkg.github.com/binxhealth/grails-spring-security-oauth2"
+			credentials {
+				username "${gprUser ?: System.getenv('GPR_USER')}"
```

Comment:
Can you switch these two? The env var should take precedence here. In fact, this will error if you do not define the gradle properties even if you have the env var.